client/components/product-icon/README.md

```diff
@@ -0,0 +1,59 @@
+ProductIcon Component
+=============
+
+ProductIcon component is a React component to display plans or products icons in SVG format.
+
+We have four different plans icons:
+- Free
+- Personal
+- Premium
+- Business
+
+Jetpack plans also use these plans icons.
+
+There is also one product icon:
+- Jetpack Backup
+
+## Usage
+
+```jsx
+import ProductIcon from 'components/product-icon';
+import { PLAN_FREE, PLAN_BUSINESS } from 'lib/plans/constants';
+import { PRODUCT_JETPACK_BACKUP_DAILY } from 'lib/products-values/constants';
+
+export default function MyComponent() {
+    return (
+        <div className="my-component">
+            <ProductIcon product={ PLAN_FREE } />
+            <ProductIcon product={ PLAN_BUSINESS } />
+            <ProductIcon product={ PRODUCT_JETPACK_BACKUP_DAILY } />
+        </div>
+    );
+}
+
+```
+
+## Props
+
+### `product`
+
+Plan constant from `lib/plans/constants` or product constant from `lib/products-values/constants`. Can be one of:
+
+- PLAN_FREE,
+- PLAN_PERSONAL,
+- PLAN_PREMIUM,
+- PLAN_BUSINESS,
```

Comment:
Guess we don't have icons for WP.com blogger or ecommerce plans, is that intentional?

Reply:
Well, actually we have icons for both. I overlooked this error when moving the code over from `PlanIcon` component.